### ISSUE TITLE
image_write (format ="svg") not working on annotated images

### ISSUE THREAD

**Issue by jcolomb** [ropensci/magick#202]
here the test:
```
tiger <- image_read_svg("http://jeroen.github.io/images/tiger.svg")

new=tiger %>%
  image_annotate("tiger", gravity = "South", size = 12, 
                 location = "+0+15",
                 color = 'black',boxcolor = "white", degrees = -10)

magick::image_write(new, path ="test.html", format ="svg", flatten = TRUE)
```
Note that` magick::image_write(tiger, path ="test.html", format ="svg", flatten = TRUE)` works, and that a png write also works.

**Comment by jcolomb** [comment]
Got a workaround by writing a png file, reading that file and saving afterwards:
```
magick::image_write(new, path =paste0("temp.png" ), format ="png", flatten = TRUE)
new2=image_read("temp.png")

magick::image_write(new2, path =paste0("tiger" ), format ="svg")
```

**Comment by jeroen** [comment]
Once you modify an image, imagemagick converts it internally to a raster. So I don't think you can save it to real svg anymore.

I think your workaround ends up saving at as a raster svg data, rather than the vectors. 

Either way this is something in imagemagick itself, not something I can control in the R wrapper.

**Comment by jcolomb** [comment]
Saving a raster svg should still work when using image_write() ? That is a R isuue probably?

Am 19. Juli 2019 16:44:27 MESZ schrieb Jeroen Ooms <notifications@github.com>:
>Closed #202.
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/ropensci/magick/issues/202#event-2496070556

-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9 Mail gesendet.
